tests/TestCase/Database/Schema/MysqlSchemaTest.php

```diff
@@ -1075,7 +1075,7 @@ public function testCreateSql()
                 'type' => 'json',
             ])
             ->addColumn('hash', [
-                'type' => 'string',
+                'type' => 'char',
                 'fixed' => true,
```

Comment:
I was writing the docs up and I'm wondering what the behavior should be when a fixture has `type => string, fixed => true` now? Does the generated column end up as a CHAR or a VARCHAR? Should this scenario emit an error, does the `fixed` flag even make sense anymore?

Reply:
CHAR _implies_ that the column is fixed. I think having a flag should not make sense anymore but, I am not sure about the users. Is it intuitive enough...? Well, I am confused...

Reply:
Could we remove the `fixed` flag? I agree it is confusing especially in the context of both a `string` and `char` type.

Reply:
Now that we have a separate type itself for fixed length columns having the `fixed` flag doesn't make sense.

Reply:
Cool!!! Will do it this weekend...... 🤘